gsg-kit.md

```diff
@@ -347,7 +332,7 @@ The sample client application is called OTGC (Onboarding Tool and Generic Client
 **[Todo OCF: This section needs verification and some detail where noted]**
 
 
-**[It should not be the same example as in the SW Simulation tutorial. It should be device-relevant.]
+**[It should not be the same example as in the SW Simulation tutorial. It should be device-relevant. If they have the Explorer Hat, they can observe pressing the buttons]
```

Comment:
remove comment

Reply:
The explorer hat buttons stuff is in step 10 above.